### ISSUE TITLE
Huge Subtitles

### ISSUE THREAD

**Issue by kanchev** [jellyfin/jellyfin-chromecast#56]
**Describe the bug**
No matter the subtitles settings in Jellyfin the subtitles on chromecast are always huge. This makes the usage of subtitles on chromecast practically useless since they are covering a lot of the video. On big TVs it's also not practical to read the huge letters.

**To Reproduce**

1. Login to Jellyfin, e.g. via Web or Android
2. Change your preferred subtitles size via profile settings
3. Cast a movie with subtitles
4. No matter the preferred subtitles size the displayed subtitles are always the same (huge)

**Expected behavior**

The subtitles settings should be adjustable similarly to jellyfin's web interface. Many movies have a black stripe on the bottom (and top) of the screen which could be used for the subtitles. Depending on the height of the black stripe the user should be allowed to change the size of the subtitles so that they can fit in.

**System (please complete the following information):**
 - OS: Android App 1.0.2
 - Browser: Chrome
 - Jellyfin Version:  10.6.3
 - Cast client: Ultra

**Comment by YouKnowBlom** [comment]
Does it happen only on external SRT subs?

**Comment by kanchev** [comment]
I'm using mostly external SRT subs but it's the same with integrated (.mpeg) subs.

**Comment by YouKnowBlom** [comment]
Does the text have a black background behind it?

**Comment by stardnce** [comment]
Confirmed. Subtitles are huge with Chromecast (ultra) and Android app. Adjusting subtitle size from app does not work. Issue confirmed with external subs, haven't tried with embedded. Is this a Chromecast ultra-only issue?

**Comment by grglzrv** [comment]
yeah the same. the subs are very big and there are spaces between each characters

**Comment by ferferga** [comment]
Can we have an screenshot please?

**Comment by grglzrv** [comment]
> Can we have an screenshot please?

Yes, the subs are Bulgarian (external .srt file ) the name of the subs are with the same name like the movie.

**Comment by grglzrv** [comment]
![Screenshot_2021-11-17-21-03-16-911_com miui gallery](https://user-images.githubusercontent.com/19932357/142265742-2770aff0-23f3-47cb-8238-40f69ba7ab8a.jpg)
![Screenshot_2021-11-17-21-03-06-815_com miui gallery](https://user-images.githubusercontent.com/19932357/142265748-71a1787d-ba45-4cb2-8997-4f42a5db15f3.jpg)
![Screenshot_2021-11-17-21-02-51-383_com miui gallery](https://user-images.githubusercontent.com/19932357/142265753-0b8ad83f-58e7-41f5-b977-7139643d5bab.jpg)
![Screenshot_2021-11-17-21-02-36-188_com miui gallery](https://user-images.githubusercontent.com/19932357/142265756-e5747b30-f316-49c4-8ae7-514261e9922e.jpg)

**Comment by grglzrv** [comment]
I stream the movie from my phone via the jellyfin android app, subs settings are :
![Screenshot_2021-11-17-21-04-41-199_org jellyfin mobile](https://user-images.githubusercontent.com/19932357/142266025-de9fc2bd-d8e6-4a91-9df6-308292893b8a.jpg)
![Screenshot_2021-11-17-21-04-29-464_org jellyfin mobile](https://user-images.githubusercontent.com/19932357/142266030-96ca3219-9613-4d7d-a19d-09b5d17e34e0.jpg)

**Comment by grglzrv** [comment]
I dont have this issue with Emby server with the same settings

**Comment by grglzrv** [comment]
With Chrome i dont have this issue
![Screenshot_2021-11-17-21-16-22-713_com miui gallery](https://user-images.githubusercontent.com/19932357/142267593-a47556db-4874-4a82-bb20-8360ea7fb30f.jpg)

**Comment by grglzrv** [comment]
Jellyfin version is 10.8.0, docker image is linuxserver/jellyfin:2021.11.17, you may find the attached bg subs
[arcane_e5.zip](https://github.com/jellyfin/jellyfin-chromecast/files/7557429/arcane_e5.zip)
s

**Comment by ferferga** [comment]
But in the Chromecast client please

**Comment by grglzrv** [comment]
> But in the Chromecast client, please

i did not get that, what kinda info is needed? I use the android app -  Jellyfin in order to cast the movie from my phone Xiaomi Mi 10

**Comment by ferferga** [comment]
A photo showing the huge subtitles in your TV (with Chromecast)

The sending app (or casting to the Android TV app using JF casting) is a completely separate issue

**Comment by grglzrv** [comment]
> A photo showing the huge subtitles in your TV (with Chromecast)
> 
Yes, i casted from my phone to Samsung Tv/monitor  with Google Chromecast 2.0 device, i dont have this issue with Samsung 40" , inly with 27"
